lib/buildSystem.js

```diff
@@ -41,6 +41,11 @@ BuildSystem.prototype._ensureInstalled = async function () {
 };
 
 BuildSystem.prototype._showError = function (e) {
+    if (this.log === undefined) {
```

Comment:
_showError is az immutable method, we cannot setup a verbose logger there. IF there is no logger, then the message should go to the console by using console.error().